packages/editor/src/components/save-publish-panels/index.js

```diff
@@ -55,6 +57,49 @@ export default function SavePublishPanels( {
 		[]
 	);
 
+	const registry = useRegistry();
+
+	useEffect( () => {
```

Comment:
I actually wonder if this should be just embedded in the regular save function. Instead of being a hook.

Reply:
Maybe, but then we again have template specific stuff in the store and actions?